autoware_adapi_v1_msgs/system/msg/MRMState.msg

```diff
@@ -0,0 +1,15 @@
+builtin_interfaces/Time stamp
+
+uint16 UNKNOWN = 0
+
+uint16 NORMAL = 1
+uint16 MRM_OPERATING = 2
+uint16 MRM_SUCCEEDED = 3
+uint16 MRM_FAILED = 4
+
+uint16 NONE = 101
```

Comment:
What is the reason for using 3 digits for this?

Reply:
When you put a behavior variable in the state, it will work unintentionally. And vice versa.

Reply:
Seeing the existing other messages, it seems they are not so.
https://github.com/autowarefoundation/autoware_adapi_msgs/blob/ac51d9cd88154dfc3d1de8ef8a0b3b4658bfee57/autoware_adapi_v1_msgs/planning/msg/SteeringFactor.msg

@isamu-takagi What do you think?

I think we should define the guidelines for this rule.

Reply:
[My first idea](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/#constants-and-enumeration) was as @mkuri -san said. However, as a result of implementation, I think the following is good.
- Group by definition
- Assign numbers to each group (allow duplication)
- Common group for shared constants (e.g. UNKNOWN, NONE)

Reply:
@mkuri @kenji-miyake 
Applying the above policy will look like this. https://github.com/autowarefoundation/autoware_adapi_msgs/pull/16#discussion_r1017582282

Reply:
Fix constants with the following commit.
https://github.com/autowarefoundation/autoware_adapi_msgs/pull/16/commits/9f86c2814267f8c07a09597c7558e18615b4ef7e